scripts/fork/helpers/runPoolLifecycle.js

```diff
@@ -60,31 +66,31 @@ async function runPoolLifecycle (prizePool, signer) {
 
   green(`New ticket balance: ${ethers.utils.formatUnits(ticketBalance, decimals)}`)
 
-  dim(`Starting award...`)
-  await prizeStrategy.startAward()
-  await increaseTime(1)
-  dim(`Completing award...`)
-  const awardTx = await prizeStrategy.completeAward()
-  const awardReceipt = await ethers.provider.getTransactionReceipt(awardTx.hash)
-  const awardLogs = awardReceipt.logs.map(log => { try { return prizePool.interface.parseLog(log) } catch (e) { return null }})
-  const awarded = awardLogs.find(event => event && event.name === 'Awarded')
-
-  if (awarded) {
-    console.log(`Awarded ${ethers.utils.formatUnits(awarded.args.amount, decimals)} token`)
-  } else {
-    console.log(`No prizes`)
-  }
+  // dim(`Starting award...`)
```

Comment:
Commented out because the RNG provider needs to be changed for this to work

Reply:
Will refactor this function into two pieces - depositAndWithdraw and startAndComplete